assimilation_code/modules/assimilation/normal_distribution_mod.f90

```diff
@@ -285,14 +287,17 @@ function inv_std_normal_cdf(quantile) result(x)
 ! Given a quantile q, finds the value of x for which the standard normal cdf
 ! has approximately this quantile
 
-! Where should the stupid p type come from
 type(distribution_params_type) :: p
 real(r8) :: mean, sd
 
 ! Set the mean and sd to 0 and 1 for standard normal
 mean        = 0.0_r8;     sd          = 1.0_r8
 p%params(1) = mean;       p%params(2) = sd
 
+! Normal is unbounded
+p%bounded_below = .false.;       p%bounded_above = .false.
+p%lower_bound   = missing_r8;    p%upper_bound   = missing_r8
+
```

Comment:
not setting the distribution type here:

`p%distribution_type = NORMAL_DISTRIBUTION`

Reply:
set in 7930bff444afa473dc22838fc6029c71dc7b666e